iot-hub/Samples/device/MethodSample/MethodSample.cs

```diff
@@ -59,6 +59,17 @@ public async Task RunSampleAsync()
             {
                 await Task.Delay(1000);
             }
+
+            // This is how one can unsubscribe a callback for the methods using a null callback handler

```

Comment:
nit: maybe reword: "You can unsubscribe from receiving a callback for direct methods by setting a null callback handler"